Modules/Loadable/Terminologies/Widgets/qSlicerTerminologyNavigatorWidget.cxx

```diff
@@ -967,11 +973,36 @@ QString qSlicerTerminologyNavigatorWidget::nameFromTerminology(vtkSlicerTerminol
   // Assemble from selection if there is no label
   if (name.isEmpty())
   {
-    name = entry->GetTypeObject()->GetCodeMeaning();
-
+    // Set type and modifier in name
     if (entry->GetTypeObject()->GetHasModifiers() && entry->GetTypeModifierObject() && entry->GetTypeModifierObject()->GetCodeValue())
     {
-      name += QString(", %1").arg(entry->GetTypeModifierObject()->GetCodeMeaning());
+      //: For formatting of terminology entry with a modifier. %1 is structure name (e.g., "Kidney"), %2 is modifier (e.g., "Left")
```

Comment:
Why the colon? `//:`

Reply:
`//:` is the syntax for message to translators, shown in Weblate and Qt Linguist, for the message in the line below.